<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCodeTool</class>
 <widget class="QWidget" name="frmCodeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labPath">
       <property name="text">
        <string>路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>H:/codeniubi/quc/designer/plugin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>选择目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labName">
       <property name="text">
        <string>类名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>GaugeCar1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>一键搞定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtHeader"/>
   </item>
   <item>
    <widget class="QTextEdit" name="txtSource"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtPath</tabstop>
  <tabstop>btnSelect</tabstop>
  <tabstop>txtName</tabstop>
  <tabstop>btnDo</tabstop>
  <tabstop>txtHeader</tabstop>
  <tabstop>txtSource</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
